package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sierra-library",
-  "version": "3.0.11",
+  "version": "3.0.11-development",
```

Comment:
Are you sure to merge this to master with the version like this? (just asking, dunno your plans at all :wink:)